Modules/IO/NIFTI/src/itkNiftiImageIO.cxx

```diff
@@ -2030,9 +2030,9 @@ NiftiImageIO::SetImageIOOrientationFromNIfTI(unsigned short dims, double spacing
           rotation.set_column(i, rotation.get_column(i).normalize());
         }
 
-        // Only orthonormal matricies have transpose as inverse
-        const vnl_matrix_fixed<float, 3, 3> candidate_identity = rotation * rotation.transpose();
-        const bool                          is_orthonormal = candidate_identity.is_identity(1.0e-4);
+          // Only orthonormal matrices have transpose as inverse
+          const vnl_matrix_fixed<float, 3, 3> candidate_identity = rotation * rotation.transpose();
+          const bool                          is_orthonormal = candidate_identity.is_identity(1.0e-4);
```

Comment:
Was whitespace added in the rebase or was it an error on my part ?

Reply:
I ran `pixi run pre-commit-run -a clang-format`, updated commit message to keep second line blank, and force-pushed it to your branch. That affected 3 additional files.

Sidenote: I could not do a normal commit with you as the author, as some pre-commit hook wanted a full name, not a user ID as author.

Reply:
That's fine with me. I don't need attribution. Sorry for all the extra work. I was hoping it would be low-maintenance.

Reply:
That wasn't hard to do. Thank you for contributing this.